internal/cmd/kafka/command_acl_delete.go

```diff
@@ -49,7 +50,19 @@ func (c *aclCommand) delete(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
-	_, userIdMap, err := c.mapResourceIdToUserId(nil)
+	serviceAccounts, err := c.Client.User.GetServiceAccounts(context.Background())
+	if err != nil {
+		return err
+	}
+
+	adminUsers, err := c.Client.User.List(context.Background())
+	if err != nil {
+		return err
+	}
+
+	users := append(serviceAccounts, adminUsers...)
```

Comment:
This is repeated 3 times! Let's make a helper function and put it in the top-level command_acl.go file?